cmd/machine-config-daemon/pivot.go

```diff
@@ -22,12 +22,12 @@ import (
 
 // flag storage
 var keep bool
-var reboot bool
 var fromEtcPullSpec bool
 
 const (
-	etcPivotFile       = "/etc/pivot/image-pullspec"
-	runPivotRebootFile = "/run/pivot/reboot-needed"
+	// etcPivotFile is used for 4.1 bootimages and is how the MCD
+	// currently communicated with this service.
```

Comment:
nit: `currently communicated` would've made my English teacher's stomach turn :laughing: 
Is it `currently communicates` or `previously communicated` (previously as in before this PR)?